clouddriver-kubernetes/src/test/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/ops/KubernetesUtilSpec.groovy

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.deploy.ops
+
+import com.netflix.spinnaker.clouddriver.kubernetes.deploy.KubernetesUtil
+import spock.lang.Specification
+
+class KubernetesUtilSpec extends Specification {
+  private final String REGISTRY1 = 'gcr.io'
+  private final String REGISTRY2 = 'localhost:5000'
+  private final String REPOSITORY1 = 'ubuntu'
+  private final String REPOSITORY2 = 'library/nginx'
+  private final String TAG1 = '1.0'
+  private final String TAG2 = 'mytag'
+
+  void "should correctly build an image description 1"() {
```

Comment:
All of these test cases can probably be combined into one that uses `@Unroll` and `where:`.